torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -811,6 +833,9 @@ class IrParser {
   }
 
   static bool querySkipSymbolSet(c10::Symbol symbol, bool flip) {
+    initRegistry();
```

Comment:
This is a small fix for skip symbol registration. It has been pushed to upstream in a separate PR